### ISSUE TITLE
Error Make failed with exit code 2, Node 13

### ISSUE THREAD

**Issue by deemalpatel** [Blizzard/node-rdkafka#733]
**Environment Information**
 - OS [Mac Mojave 10.14.6]: 
 - Node Version [13.2.0]:
 - NPM Version [6.13.1]:
 - node-rdkafka version [2.4.2]:

**Steps to Reproduce**

I am trying install rdkafka@2.4.2 on my project and every time it breaks with 'Error Make failed with exit code 2.' I already have openssl installed via brew and I added the LDFLAGS and CPPFLAGS to my bash profile already but I still am generating this. Anyone have this same issue and was able to solve it? 

**node-rdkafka Configuration Settings**

**Additional context**

gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (events.js:210:5)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:270:12)
gyp ERR! System Darwin 18.7.0
gyp ERR! command "/usr/local/Cellar/node/13.2.0/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/cn193394/Click/click-taskvalidator/node_modules/node-rdkafka
gyp ERR! node -v v13.2.0
gyp ERR! node-gyp -v v5.0.5
gyp ERR! not ok 
npm WARN click-taskvalidator@1.0.4 No repository field.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! node-rdkafka@2.4.2 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the node-rdkafka@2.4.2 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:

**Comment by analytik** [comment]
node-rdkafka@2.4.2 is a very old version, it will never build. Your best bet is using Node.js v12 and node-rdkafka@2.7.x - it's unlikely it will build on Node.js v13, but feel free to try. If it doesn't help, you'll need to submit a pull request to add Node.js v13 compatibility, or wait for someone else does.

**Comment by ntkumar1** [comment]
Hi everyone, we experience similar issue as well.

**Environment Information**
OS [macOS Catalina 10.15.1]:
Node Version [13.8]:
NPM Version [6.13.6]:
node-rdkafka version [2.7.4]:

**Steps to Reproduce**
We are using a base image with Node Version [13.8-slim], NPM Version [6.13.6] and install 
node-rdkafka version [2.7.4]. The below error occurs during the make command.

**Logs**
```
node-librdkafka.target.mk:130: recipe for target 'Release/obj.target/node-librdkafka/src/binding.o' failed
rm 11a9e3388a67e1ca5c31c1d8da49cb6d2714eb41.intermediate
make: Leaving directory '/usr/src/app/node_modules/node-rdkafka/build'
make: *** [Release/obj.target/node-librdkafka/src/binding.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (events.js:321:20)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:275:12)
gyp ERR! System Linux 4.19.76-linuxkit
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /usr/src/app/node_modules/node-rdkafka
gyp ERR! node -v v13.8.0
gyp ERR! node-gyp -v v5.0.5
gyp ERR! not ok 
npm WARN security-advisor-shard-manager@0.0.1 license should be a valid SPDX license expression
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! node-rdkafka@2.7.4 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the node-rdkafka@2.7.4 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-02-20T10_10_01_699Z-debug.log
```

**Comment by analytik** [comment]
@ntkumar1 yes, like I said, Node.js v13 is not supported at this time. Downgrade to Node.js v12, or migrate to Kafkajs instead.

**Comment by skairali** [comment]
@analytik  Is there a plan for supporting Node V13.  Would be great to know timelines.  There are some security vulnerabilities that are fixed in Node 13.8.0 onward which we would like adopt.

**Comment by analytik** [comment]
@skairali see https://github.com/Blizzard/node-rdkafka/issues/628

There is no guarantee of anything being updated ever again. Knowing this, do what you think is best for your work.
